test/ttmlir/Dialect/TTNN/optimizer/bf_interleaved_policy/fork_join_02.mlir

```diff
@@ -0,0 +1,35 @@
+// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline="enable-optimizer=true memory-layout-analysis-enabled=true memory-layout-analysis-policy=BFInterleaved" %s | FileCheck %s
+//
+//         A
+//         |
+//         B
+//       /   \
+//      C     D
+//       \   /
+//         E
+//         |
+//         F
+//
+// There is enough L1 memory to schedule this fork-join without any DRAM spill even though
+// output tensors of C and D cannot be in L1 memory at the same time.
```

Comment:
This comment seems incorrect as it would not be possible to put everything into L1 in this case. Adjust comment.